src/components/m-datepicker-body/index.scss

```diff
@@ -0,0 +1,64 @@
+@import "../../styles/settings/colors";
+
+$datepicker-width: 309px !default;
+
+$datepicker-padding-left-right: 30px !default;
```

Comment:
I suggest call it `$datepicker-padding-x` (for x-axis and y for y-axis in case of top/bottom)

Reply:
agree

Reply:
done